5.1/config.mk

```diff
@@ -0,0 +1,3 @@
+export KIBANA_VERSION="5.1.2"
+export KIBANA_DOWNLOAD="https://artifacts.elastic.co/downloads/kibana/kibana-${KIBANA_VERSION}-linux-x86_64.tar.gz"
```

Comment:
Remove the quotes here: this is a makefile (not a shell script), so quotes are interpreted as text.

Also, the syntax for interpolation in a Makefile is `$(KIBANA_VERSION)`, not `${KIBANA_VERSION}`.

Overall, this should be:

```
export KIBANA_VERSION = 5.1.2
export KIBANA_DOWNLOAD = https://artifacts.elastic.co/downloads/kibana/kibana-$(KIBANA_VERSION)-linux-x86_64.tar.gz
export KIBANA_SHA1SUM = ba355d63fef6702109ebdbf72d7ebca0451ed7ae
```